addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -36,6 +37,11 @@ export class Chatter extends Component {
         this.onScrollDebounced = useThrottleForAnimation(this.onScroll);
         useChildSubEnv(this.childSubEnv);
 
+        onWillDestroy(() => {
+            if (this.state.thread) {
+                this.store.env.services["bus_service"].deleteChannel(this.state.thread.busChannel);
+            }
+        })
```

Comment:
Would be best to put it as side-effect of `thread_model.isDisplayedOnUpdate`, so that this also takes into account chatter in chat window (this one doesn't use `<Chatter>` component)